Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTableView.h

```diff
@@ -39,6 +39,7 @@ namespace AzToolsFramework
 
             void setModel(QAbstractItemModel *model) override;
             void SetName(const QString& name);
+            QModelIndex GetHoveredIndex() const { return model()->index(m_hoveredRow, m_hoveredColumn);}
```

Comment:
althought I like this, I believe the coding standards prohibit implementation in the same line